src/items/equipment/cunning.json

```diff
@@ -0,0 +1,137 @@
+{
+  "_id": "Z0kVIxKfWNcXeFDb",
+  "name": "Cunning",
+  "type": "fusion",
+  "img": "systems/sfrpg/icons/equipment/weapons/weapon-fusions-1.webp",
+  "system": {
+    "type": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 6
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "-",
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p><span>When you strike a creature with a cunning weapon and deal damage, you can attempt an [[/r 1d20+@skills.eng.mod]]{Engineering}, [[/r 1d20+@skills.lsc.mod]]{Life Science}, or [[/r 1d20+@skills.mys.mod]]{Mysticism} check to identify the creature as appropriate (this takes no action). You gain a +2 circumstance bonus to this check. If you succeed, in addition to learning useful information about the creature, you gain a +1 circumstance bonus to the next saving throw you must attempt against that creature’s abilities within the next minute.</span></p>"
```

Comment:
Use `@Check` instead. E.g `@Check[type:life-science]`

Also this html is wrapped in a span tag, delete that